context/app/static/js/pages/Template/Template.tsx

```diff
@@ -81,7 +101,17 @@ function ExampleAccordion({ example, templateKey, defaultExpanded }: ExampleAcco
                 </LabelledSectionText>
               )}
             </Stack>
-            <WorkspaceDatasetsTable datasetsUUIDs={[...datasets]} isSelectable={false} />
+            <WorkspaceDatasetsTable
```

Comment:
Do we want to track this across `WorkspaceDatasetsTable`? Should we move `trackEvent` inside of the `WorkspaceDatasetsTable` and pass the differing tracking details?